packages/backend/src/graphql/schema.graphql

```diff
@@ -411,7 +411,8 @@ enum NotificationFrequency {
 
 input UpdateFlowConfigInput {
   id: String!
-  notificationFrequency: NotificationFrequency!
+  notificationFrequency: NotificationFrequency
+  onFirstLoad: Boolean
```

Comment:
nit: hasLoadedOnce

Reply:
roger boss

Reply:
ok means im gonna have to negate everything to the opposite